doc/classes/Parallax2D.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="Parallax2D" inherits="Node2D" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+		A node used to create a parallax scrolling background.
+	</brief_description>
+	<description>
+		A Parallax2D is used to create a parallax effect. It can move at a different speed relative to the camera movement using [member scroll_scale]. This creates an illusion of depth in a 2D game. If manual scrolling is desired, the [Camera2D] position can be ignored with [member override_ignore_camera_scroll].
+		[b]Note:[/b] Any changes to this node's position made after it enters the scene will be ignored.
+	</description>
+	<tutorials>
+	</tutorials>
+	<members>
+		<member name="autoscroll" type="Vector2" setter="set_autoscroll" getter="get_autoscroll" default="Vector2(0, 0)">
+			When set, the offset will scroll automatically by the set amount of pixels per second.
+		</member>
+		<member name="limit_begin" type="Vector2" setter="set_limit_begin" getter="get_limit_begin" default="Vector2(0, 0)">
+			Top-left limits for scrolling to begin. If the camera is outside of this limit, the Parallax2D will stop scrolling. Must be lower than [member limit_end] to work.
+		</member>
+		<member name="limit_end" type="Vector2" setter="set_limit_end" getter="get_limit_end" default="Vector2(0, 0)">
+			Bottom-right limits for scrolling to end. If the camera is outside of this limit, the Parallax2D will stop scrolling. Must be higher than [member limit_begin] to work.
+		</member>
+		<member name="offset" type="Vector2" setter="set_offset" getter="get_offset" default="Vector2(0, 0)">
+			The Parallax2D's offset relative to its parent.
+		</member>
+		<member name="override_follow_viewport" type="bool" setter="set_follow_viewport" getter="get_follow_viewport" default="true">
+			If [code]true[/code], Parllax2D will follow along with the current camera. It may be desired to turn off if the Parallax2D is in a [CanvasLayer] separate from the current camera.
+		</member>
+		<member name="override_ignore_camera_scroll" type="bool" setter="set_ignore_camera_scroll" getter="is_ignore_camera_scroll" default="false">
```

Comment:
This property is confusing. It says "ignore", but the parallax will be affected. Also isn't it the same as above?

Reply:
They're not the same, but if it's outwardly confusing, it's worth reworking (also there's a typo). I was curious if anyone else had a better name for these concepts.

`Parallax2D` works by updating its position based on the position of the camera and offsetting it from there (otherwise the repeat wouldn't work). Just like in `ParallaxBackground`, it may be desired to not use a camera or to have a custom one, where you don't want the current camera to update the `Parallax2D`'s position (if you have one), but you still want a parallax effect. `override_ignore_camera_scroll` means that when you the camera scrolls, its position will not be propagated to the `Parallax2D` node. From there you can supply an `override_screen_offset` value, and it will honor that instead.

Of course, any rotation of the current camera will still occur since it's a `Node2D`. If the camera's rotation is not desired, you can always put it in a separate `CanvasLayer` like `ParallaxBackground` was. However, if you put a `Parallax2D` in a separate `CanvasLayer` and you still want the position to match the camera, the camera position being provided will still be as if they're in the same `CanvasLayer`. To fix this, `follow_viewport` can be turned off, so the screen offset is removed after the parallax position is calculated. Similar to the `CanvasLayer` `follow_viewport` property.

Any suggestions?

Reply:
Updated it, but let me know if there's better wording.